tap_hubspot/hubspot.py

```diff
@@ -197,20 +137,17 @@ def get_contacts(
             properties,
         )
 
-        for chunk in chunker(gen, 10):
+        for chunk in chunker(gen, 100):
             ids: List[str] = [contact["id"] for contact in chunk]
 
-            deals_associations = self.get_associations(obj_type, "deals", ids)
             companies_associations = self.get_associations(obj_type, "companies", ids)
 
             for i, contact_id in enumerate(ids):
                 contact = chunk[i]
 
-                deals = deals_associations.get(contact_id, [])
                 companies = companies_associations.get(contact_id, [])
 
                 contact["associations"] = {
-                    "deals": {"results": deals},
```

Comment:
we should split these into separate commits as well, or fixup them into the initial one.